app/src/main/java/com/nervousfish/nervousfish/data_objects/Contact.java

```diff
@@ -11,15 +11,22 @@
 import java.util.Arrays;
 import java.util.Collection;
 import java.util.List;
+import java.util.Objects;
+
+import nl.tudelft.ewi.ds.bankver.IBAN;
 
 /**
  * A Java object representation (POJO) of a contact.
  */
+@SuppressWarnings({"PMD.UselessParentheses", "PMD.NullAssignment"})
+//1. The parentheses on line 125 are not useless.
+//2. We want iban to be immutable, but not a required field.
```

Comment:
Are you sure? The IBan seems to be a required field in the `CreateProfileActivity`...